### ISSUE TITLE
No prop available to change aria-autocomplete of react-select Input

### ISSUE THREAD

**Issue by integrellaabhijeet** [JedWatson/react-select#4950]
There is no prop available for react select to change aria-autocomplete attribute like we can change aria-label.
The value for aria-autocomplete is set to list and is unable to change through props. 

Refer the following url to the line of code.
https://github.com/JedWatson/react-select/blob/master/packages/react-select/src/Select.tsx#L1560

Because of aria-autocomplete="list", the screen reader reads out things that the user shouldn't really need to hear. something like "edit text autocompletion lists" is read out.

to prevent that we need to set aria-autocomplete to none. But since it has constant value "list", we are not able to do so.

**Comment by bahadurharris** [comment]
I am also facing a similar issue where screen reader is announcing "Drop Down combo box collapsed has auto complete" because of the aria-autocomplete="list" where as I've set the isSearchable prop to false and I also tried to change the value of this none but it didn't worked, please check and update asap

**Comment by bahadurharris** [comment]
![select drop down has autocomplete](https://github.com/JedWatson/react-select/assets/99205086/6e0c5221-d3d6-4ad2-bf4a-6915a462a886)
